client/me/security-2fa-disable/index.jsx

```diff
@@ -87,14 +87,34 @@ class Security2faDisable extends Component {
 		}
 
 		return (
-			<p>
-				{ translate(
-					'Your account is currently protected by Two-Step ' +
-						'Authentication. While enabled, logging in to WordPress.com ' +
-						'requires you to enter a unique passcode, generated by an app on ' +
-						'your mobile device, in addition to your username and password.'
-				) }
-			</p>
+			<div>
+				<p>
+					{ translate(
+						'Your account is currently protected by Two-Step ' +
+							'Authentication. While enabled, logging in to WordPress.com ' +
+							'requires you to enter a unique passcode, generated by an app on ' +
+							'your mobile device, in addition to your username and password.'
+					) }
+				</p>
+				<p>
+					{ this.props.translate(
+						'If you are planning on switching to a new device, you will want to ' +
+							'{{changephonelink}}follow these steps{{/changephonelink}} to ' +
+							'avoid being accidentally locked out of your user account.',
```

Comment:
Thanks! I'll use the latter because it has the gist right in the first word and that should help when skimming through the page.

>still have a bit of copy-envy over how clear and simple Google's `Change Phone` is.

Same. I tested making a secondary action button next to "disable" button but it didn't feel like an external link so I went this route instead.

We should probably use a button + short label once we can do more than just redirect to a help page. :-)